watch_service.go

```diff
@@ -140,24 +142,23 @@ const (
 )
 
 // WatchListener listens to Watcher.
-type WatchListener func(revision int, value interface{})
+type WatchListener func(v WatchResult)
```

Comment:
`v` -> `result` or `watchResult`? Is it conventional to use such a short variable name for callback parameters?

Reply:
Yes, it's convenient because we have only one param here. And `v` is also common in golang. But I agree that we could make it more clearly with `result` or `watchResult`.
Thank you ^_^